demo/views/demo-all.dust

```diff
@@ -46,14 +46,11 @@
   <input type="text" class="offleft">
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
   <!-- Disable Auto Init with this flag -->
   <!-- true = JavaScript will not initialize automatically -->
   <!-- false = JavaScript will initialize automatically -->
-  <script>
-    CarbonComponents.settings.disableAutoInit = false;
```

Comment:
Seems that this line is for [L51](https://github.com/carbon-design-system/carbon-components/pull/143/files#diff-627e32a3f2084e6bc734d3d244058336R51)-[L53](https://github.com/carbon-design-system/carbon-components/pull/143/files#diff-627e32a3f2084e6bc734d3d244058336R53) explanations, so we have two options here:

1. Removing [L51](https://github.com/carbon-design-system/carbon-components/pull/143/files#diff-627e32a3f2084e6bc734d3d244058336R51)-[L53](https://github.com/carbon-design-system/carbon-components/pull/143/files#diff-627e32a3f2084e6bc734d3d244058336R53) altogether
2. Keeping this line